File: C_TUT/tut5.c
#include <stdio.h>

// STRINGs

char * name = "Connor Brady";

char correct_name[] = "Connor Brady";

int age = 18;

printf("%s is %d years old.\n", name, age);
printf("%d\n", strlen(name));

char* newname = "Connor";

if(strncmp(newname, "Connor", 6) == 0) {
	printf("Hello, Connor!\n");
} else {
	printf("You are not Connor.\n");
}

File: PY_TUT/Tut5.py
# String formatting

# %s - String (or any object with a string representation, like numbers)

# %d - Integers

# %f - Floating point numbers

# %.<number of digits>f - Floating point numbers with a fixed amount of digits to the right of the dot.

# %x/%X - Integers in hex representation (lowercase/uppercase)

name = "Connor"
age = 18;

print("Hello, %s!" % name)
print("%s is %d years old." % (name, age))